src/app/users/user-layout/home/home.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../../../services/auth.service';
+import { MenuService } from 'src/app/services/menu.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css'],
+})
+export class HomeComponent implements OnInit {
+  isSignedIn = false;
+  itemName!: string;
+  price!: number;
+  image!: Promise<string | void>;
```

Comment:
remove ! and copy the error then check the solution. there might be the way to workaround where you can skip this !